### PR TITLE
Refine file associations

### PR DESCRIPTION
**Pull request by MetRonnie**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,9 +25,10 @@
                     "cylc"
                 ],
                 "filenamePatterns": [
-                    "suite*.rc",
+                    "*suite*.rc",
                     "suite*.rc.*",
                     "*.cylc",
+                    "*.cylc.*",
```
Comment (wxtim) [inline package.json:31]:
```suggestion
"*.cylc*",
```

Comment (wxtim) [reply package.json:31]: Are these globs rather than regexes?

**Review by wxtim — APPROVED** [review]
I've made 1 comment - feel free to use or ignore, and to merge having done so.

File: package.json (hunk 1 above)
Comment (MetRonnie) [reply package.json:31]: Was trying to look this up but then realised/remembered that `filenamePatterns` isn't actually documented! But yes, it's globs.

Comment (MetRonnie) [reply package.json:31]: Are you aware of any uses of the file pattern `*.cylc*` without a second `.`? If so I'll make the change (and I guess `*suite.rc.* -> *suite.rc*`), but if not then I won't bother 😛

**Pull request merged**